keydb/templates/secret-utils.yaml

```diff
@@ -8,7 +8,7 @@ type: Opaque
 stringData:
   server.sh: |
     #!/bin/sh
-    set -euxo
+    set -euo pipefail
```

Comment:
We might want to log the generated command-line arguments with REDIS_PASSWORD as ´***`

Reply:
Now it looks like this:
```sh
$ kubectl logs -n henri pod/keydb-0

Defaulted container "keydb" out of: keydb, redis-exporter
keydb-server /etc/keydb/redis.conf --active-replica yes --multi-master yes --appendonly no --bind 0.0.0.0 --port 6379 --protected-mode no --server-threads 2 --masterauth *** --requirepass *** --replicaof keydb-1.keydb-headless 6379 --replicaof keydb-2.keydb-headless 6379
1:1:C 29 Jun 2023 09:10:43.541 * Notice: "active-replica yes" implies "replica-read-only no"
```

Reply:
And without REDIS_PASSWORD
```sh
$ kubectl logs -n henri pod/keydb-0

Defaulted container "keydb" out of: keydb, redis-exporter
keydb-server /etc/keydb/redis.conf --active-replica yes --multi-master yes --appendonly no --bind 0.0.0.0 --port 6379 --protected-mode no --server-threads 2 --replicaof keydb-1.keydb-headless 6379 --replicaof keydb-2.keydb-headless 6379
1:1:C 29 Jun 2023 09:28:36.138 * Notice: "active-replica yes" implies "replica-read-only no"
```